### DISCUSSION TITLE
Video is getting cropped at right side.

### DISCUSSION THREAD

**Discussion by suryakrishna1524** [tamius-han/ultrawidify#239] [General]
I had enabled the automatic crop in the extension and video got perfect fit in top, bottom and left but cropped away only right side. Can you suggest which thing should I change to correct this?
![Screenshot_2023-12-26-13-27-31-35_32cba261fc08a501ce70de75bdbe2575](https://github.com/tamius-han/ultrawidify/assets/75824153/f523982d-4401-40e8-886b-5e817b034ed0)
![Screenshot_2023-12-26-13-29-36-60_32cba261fc08a501ce70de75bdbe2575](https://github.com/tamius-han/ultrawidify/assets/75824153/35d8311a-3b3c-476d-a730-483b81813cf5)

**Comment by tamius-han** [comment]
Try playing around with this option under 'advanced' settings on the site that's giving you this issue (I'm only doing this on youtube for illustrative purposes): 

![slika](https://github.com/tamius-han/ultrawidify/assets/12505802/4bc36aa4-5318-470f-8bb8-82b025a63895)

Start with 1, reload page, see if it helped. If it didn't, increase the number by 1. Reload. See if the issue is fixed.

Usually, a value between 1 and 10 should ensure appropriate crop.
